packages/runc/0001-do-not-label-dev-mqueue.patch

```diff
@@ -0,0 +1,51 @@
+From 2e6701d7f47c7cd8e75beb3c8669a2d6700008e2 Mon Sep 17 00:00:00 2001
+From: Ben Cressey <bcressey@amazon.com>
+Date: Wed, 27 May 2020 23:01:54 +0000
+Subject: [PATCH] do not label /dev/mqueue
+
+The mqueue filesystem shares its label with the host, and trying to
+mount it with a different label leads to an "invalid mount" warning.
```

Comment:
Is this an artifact of our policy?  If so, I'd like to make sure we have a follow-up to convert this patch into something upstreamable.  (We can make an issue and assign it to me.)

Reply:
The same warning happens on Fedora 30, and [BZ#1425278](https://bugzilla.redhat.com/show_bug.cgi?id=1425278) shows it happening on an OpenShift node running a 4.18 kernel.

It's not specific in any way to our policy; it's just an error to attempt to apply a context to mqueue, and all recent kernels reject the mount.

Reply:
Got it.  Let's make sure we have an item to follow up so we can upstream the patch.